products/bash/configure-postgres.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+#******************************************************************************
+# Licensed Materials - Property of IBM
+# (c) Copyright IBM Corporation 2019. All Rights Reserved.
+#
+# Note to U.S. Government Users Restricted Rights:
+# Use, duplication or disclosure restricted by GSA ADP Schedule
+# Contract with IBM Corp.
+#******************************************************************************
+
+# PREREQUISITES:
+#   - Logged into cluster on the OC CLI (https://docs.openshift.com/container-platform/4.4/cli_reference/openshift_cli/getting-started-cli.html)
+#
+# PARAMETERS:
+#   -n : <POSTGRES_NAMESPACE> (string), Defaults to 'postgres'
+#   -u : <DB_USER> (string), Defaults to 'cp4i'
+#   -d : <DB_NAME> (string), Defaults to 'db_cp4i'
+#   -p : <DB_PASS> (string), Defaults to ''
+#
+#   With defaults values
+#     ./configure-postgres.sh
+#
+#   With overridden values
+#     ./configure-postgres.sh -n <POSTGRES_NAMESPACE> -u <DB_USER> -d <DB_NAME> -p <DB_PASS>
+
+function usage {
+  echo "Usage: $0 -n <POSTGRES_NAMESPACE> -u <DB_USER> -d <DB_NAME> -p <DB_PASS>"
+  exit 1
+}
+
+POSTGRES_NAMESPACE="postgres"
+DB_USER="cp4i"
+DB_NAME="db_cp4i"
+DB_PASS=""
+
+while getopts "n:u:d:p:" opt; do
+  case ${opt} in
+    n ) NAMESPACE="$OPTARG"
+      ;;
+    u ) DB_USER="$OPTARG"
+      ;;
+    d ) DB_NAME="$OPTARG"
+      ;;
+    p ) DB_PASS="$OPTARG"
+      ;;
+    \? ) usage; exit
+      ;;
+  esac
+done
+
```

Comment:
I think `DB_PASS` should not be optional. Make this error if `DB_PASS` is not passed or is blank.